app/serializers/v2/rule_serializer.rb

```diff
@@ -4,5 +4,10 @@ module V2
   # JSON API serialization for an OpenSCAP Rule
   class RuleSerializer < ApplicationSerializer
     attributes :ref_id, :title, :rationale, :description, :severity, :precedence
+    attribute :remediation_issue_id
+
+    def include_remediation_issue_id?
+      true if object.params[:parents].include?(Profile)
+    end
```

Comment:
This is nice for now (if it works), but I am trying to cook up something that looks nicer and easier to use.

Reply:
Now that I had a way to test if this method works, I can say it doesn't. 
I've used a similar approach as in v1. This seems to be working for now.

Reply:
Alright, we just discussed with Roman that we'll do some changes in the serialization, so this might be simplified in the future. We don't plan to change the behavior, just the underlying logic, so please write tests for this one.